app/scripts/lib/strings.js

```diff
@@ -41,7 +41,7 @@ define(function (require, exports, module) {
   t('Forgot password');
```

Comment:
Since this PR contains the real strings, you can remove the placeholders from this file.